spec/ruby/core/matchdata/begin_spec.rb

```diff
@@ -36,6 +36,18 @@
       match_data = /(.)(.)(\d+)(\d)/.match("THX1138.")
       match_data.begin(obj).should == 2
     end
+
+    it "raises IndexError if index is out of matches" do
```

Comment:
I was going to say "is out of matches" is awkward phrasing, but I see it matches the error message. Upstream recently fixed some message phrasing recently; maybe they'll be open to this one as well.

Reply:
I'd say let's use `out of bounds` in this spec description because that seems clearer, we generally don't use exception message in spec descriptions.